data/repositories.data.ts

```diff
@@ -0,0 +1,5 @@
+export const repositoriesCommon = {
+    repositoryName1 : 'test1',

```

Comment:
when creating test data, a distrustful name is needed, ex: instead of using "test1", use "testing add new repo"

Reply:
beside, the name of the object too needs to change, remove the common suffex